cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -300,19 +299,6 @@ template <typename settings> void Prover<settings>::execute_relation_check_round
     auto sumcheck = Sumcheck(multivariates, transcript);
 
     sumcheck.execute_prover();
-
-    // Add the multilinear evaluations produced by Sumcheck to the transcript.
```

Comment:
To move this into execute_prover, we have to pass a polynomial manifest. But it is desirable for sumcheck to be independent and testable without reference to the polynomial manifest (would have to mock this in the tests, etc.). How to handle? On the other hand, sumcheck already has a multivariates objects, and maybe that should learn about shifts. On the other hand, maybe the paradigm should be that modules expose functions so that all transcript interactions happen inside the prover? I lean toward the former.